build.gradle

```diff
@@ -63,6 +63,13 @@ dependencies {
     testImplementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: jUnitVersion
 
     testRuntimeOnly group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: jUnitVersion
+
+    testCompile 'org.junit.jupiter:junit-jupiter-api:5.5.1'
+    testCompile "org.testfx:testfx-junit5:4.0.16-alpha"
+    testCompile group: 'org.hamcrest', name: 'hamcrest', version: '2.1'
+    testCompile group: 'org.assertj', name: 'assertj-core', version: '3.13.2'
```

Comment:
are hamcrest and assertj used? Also, do we need to define JUnit as testCompile? (since we already defined it in our testImplementation/testRuntimeOnly)If we need to, should probably use the jUnitVersion variable instead of a string